jenkins-integration/jenkins-jobs/common/scripts/job-steps/021_install_common.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
When I cloned down this repo, the execute bit wasn't set on the `021_install_common.sh` file, causing the script to fail.  Would be good to update that.